base/server/python/pki/server/cli/audit.py

```diff
@@ -45,6 +47,95 @@ def __init__(self, parent):
         self.add_module(AuditFileFindCLI(self))
         self.add_module(AuditFileVerifyCLI(self))
 
+    @staticmethod
+    def print_audit_config(subsystem):
+
+        name = 'log.instance.SignedAudit.%s'
+
+        enabled = subsystem.config[name % 'enable'].lower() == 'true'
+
+        fileName = subsystem.config[name % 'fileName']
+        bufferSize = subsystem.config[name % 'bufferSize']
+        flushInterval = subsystem.config[name % 'flushInterval']
+
+        maxFileSize = subsystem.config[name % 'maxFileSize']
+        rolloverInterval = subsystem.config[name % 'rolloverInterval']
+        expirationTime = subsystem.config[name % 'expirationTime']
+
+        logSigning = subsystem.config[name % 'logSigning'].lower() == 'true'
+        signedAuditCertNickname = subsystem.config[name % 'signedAuditCertNickname']
+
+        print('  Enabled: %s' % enabled)
+
+        print('  Log File: %s' % fileName)
+        print('  Buffer Size (bytes): %s' % bufferSize)
+        print('  Flush Interval (seconds): %s' % flushInterval)
+
+        print('  Max File Size (bytes): %s' % maxFileSize)
+        print('  Rollover Interval (seconds): %s' % rolloverInterval)
+        print('  Expiration Time (seconds): %s' % expirationTime)
+
+        print('  Log Signing: %s' % logSigning)
+        print('  Signing Certificate: %s' % signedAuditCertNickname)
+
+
+class AuditConfigShowCLI(pki.cli.CLI):
+
+    def __init__(self, parent):
+        super(AuditConfigShowCLI, self).__init__(
+            'config-show', 'Display audit configuration')
+        self.parent = parent
+
+    def print_help(self):
+        print('Usage: pki-server %s-audit-config-show [OPTIONS]' % self.parent.parent.name)
+        print()
+        print('  -i, --instance <instance ID>       Instance ID (default: pki-tomcat).')
+        print('      --help                         Show help message.')
+        print()
+
+    def execute(self, argv):
+        try:
+            opts, _ = getopt.gnu_getopt(argv, 'i:v', [
```

Comment:
1st candidate for `argparse`? This can be a part of the effort of this: https://pagure.io/dogtagpki/issue/3072

Reply:
Yes, but I was just copying the code from an existing one. We can convert it into argparse as part of that ticket.